ui/src/markdown.tsx

```diff
@@ -118,8 +119,7 @@ export const Markdown = ({ source, compact = true }: { source: S, compact?: B })
         return false
       }
     }
-  React.useEffect(() => { import('./markdown.css') }, [])
```

Comment:
What problem was there exactly? Doesn't Vite support dynamically imported CSS?

Reply:
This question made me scratching my head for few hours and I don't have the answer.

Looks it should support dynamic import of all kind of files:
https://v4.vitejs.dev/guide/features.html#dynamic-import
https://v4.vitejs.dev/guide/features.html#css

But I couldn't get it to work for some reason so I went with other solution.